common/models.py

```diff
@@ -38,3 +38,30 @@ class User(AbstractUser, TimestampedModel):
     
     def __str__(self):
         return self.email
+
+# START_FEATURE django_storages
+# from django.conf import settings
+# from django.core.files.storage import FileSystemStorage
+# from storages.backends.s3boto3 import S3Boto3Storage
+# 
+# def get_file_storage():
+#     if not settings.DEBUG:
+#         return S3Boto3Storage(bucket=settings.AWS_STORAGE_BUCKET_NAME)
+#     else:
+#         return FileSystemStorage(location='media/uploads/')
+# 
+# def get_s3_path(instance, filename):
```

Comment:
These should be commented in and listed with `# TODO: delete me; this is just a reference example`